src/Illuminate/Foundation/Console/StorageLinkCommand.php

```diff
@@ -27,11 +28,19 @@ class StorageLinkCommand extends Command
      */
     public function handle()
     {
+        $files = $this->laravel->make('files');
```

Comment:
Why pull this line up here? 

Performance wise isn't it better to keep it below the return?

We don't need to always pull the filesystem out of the container?